tests/Microsoft.Azure.CosmosRepositoryTests/Providers/DefaultCosmosThroughputProviderTests.cs

```diff
@@ -73,5 +73,16 @@ public void GetThroughputPropertiesItemsWithConflictingManualValues()
             Assert.Contains("500", exception.Message);
             Assert.Contains("700", exception.Message);
         }
+
+        [Fact]
+        public void GetThroughputPropertiesContainerWithServerlessConfiguration()
+        {
+            _repositoryOptions.ContainerBuilder.Configure<AnotherTestItem>(
+                builder => builder.WithServerlessThroughput());
+
+            ThroughputProperties throughputProperties = _provider.GetThroughputProperties<AnotherTestItem>();
```

Comment:
Do you know if we can enable nullable reference types even tho we are on netstandard 2.0? Not one for this PR but was just curious?

Reply:
Yes, NRT is fully supported in .NET Standard 2.0